src/Cedar/SM.c

```diff
@@ -18452,38 +18452,35 @@ void SmServerDlgRefresh(HWND hWnd, SM_SERVER *p)
 	}
 
 	// Listener list update
-	if (p != NULL)
+	RPC_LISTENER_LIST t;

```

Comment:
Why was this check removed. p is being used directly inside the if condition

Reply:
there's check for NULL https://github.com/SoftEtherVPN/SoftEtherVPN/blob/master/src/Cedar/SM.c#L18372